src/runtime/virtcontainers/qemu_amd64.go

```diff
@@ -273,3 +278,129 @@ func (q *qemuAmd64) appendProtectionDevice(devices []govmmQemu.Device, firmware
 		return devices, "", fmt.Errorf("Unsupported guest protection technology: %v", q.protection)
 	}
 }
+
+// setup prelaunch attestation
+func (q *qemuArchBase) setupGuestAttestation(ctx context.Context, config govmmQemu.Config, path string, proxy string) (govmmQemu.Config, error) {
+	logger := virtLog.WithFields(logrus.Fields{
+		"subsystem":               "qemuAmd64",
+		"machine":                 q.qemuMachine,
+		"kernel-params-debug":     q.kernelParamsDebug,
+		"kernel-params-non-debug": q.kernelParamsNonDebug,
+		"kernel-params":           q.kernelParams})
+	switch q.protection {
+	case sevProtection:
+		logger.Info("SEV attestation: Pulling launch argument...")
```

Comment:
minor nit just from glancing over this: do you want to make `SEV attestation` a log field rather than explicitly printing it every time?

Reply:
Agreed - https://github.com/kata-containers/tests/tree/main/cmd/log-parser.

Reply:
Yes, I'll clean up these debug prints